openedx/features/qverse_features/registration/tests/factories.py

```diff
@@ -0,0 +1,39 @@
+"""
+Factories for QVerse registration app unit tests.
+"""
+import factory
+import factory.fuzzy
+import string
+
+from openedx.features.qverse_features.registration.models import (Department, QVerseUserProfile,
+                                                                  MAX_LEVEL_CHOICES, MAX_PROGRAMME_CHOICES)
```

Comment:
For more readability:
```suggestion
from openedx.features.qverse_features.registration.models import (
    Department, 
    QVerseUserProfile,
    MAX_LEVEL_CHOICES, 
    MAX_PROGRAMME_CHOICES
    )
```